### PR TITLE
Add "Homebrew Maintainer Meeting" post

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Write up of the February maintainer meeting. CC @Homebrew/maintainers for copy-editing and review.

### CODE REVIEW

File: _posts/2019-06-14-homebrew-maintainer-meeting.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Homebrew Maintainer Meeting
+author: MikeMcQuaid
+---
+In February 2019 the Homebrew maintainers had our first in-person meeting at and around the [FOSDEM 2019 conference](https://fosdem.org/2019/) in Brussels. Maintainers travelled as far as India and Canada in order to get face-time with each other and have high-bandwidth conversations.
```
Comment (vitorgalvao) [inline _posts/2019-06-14-homebrew-maintainer-meeting.md]: Maintainers travelled **from** as far as

File: _posts/2019-06-14-homebrew-maintainer-meeting.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Maintainers arrived on Friday and met informally to socialise and get to know each other. On Saturday we released [Homebrew 2.0.0](https://brew.sh/2019/02/02/homebrew-2.0.0/) and on Saturday and Sunday attended the various talks at the conference (and including a [Homebrew 2.0.0 talk](https://mikemcquaid.com/talks/homebrew-2.0.0/)]).
```
Comment (vitorgalvao) [inline _posts/2019-06-14-homebrew-maintainer-meeting.md]: I’d remove the “and” from “(**and** including a”.

**Comment by vitorgalvao** [comment]
I really like this. It’s succinct and feels personal, exposing we’ve had a good time and produced something of it, thanks to the support of the community.

File: _posts/2019-06-14-homebrew-maintainer-meeting.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+title: Homebrew Maintainer Meeting
+author: MikeMcQuaid
+---
+In February 2019 the Homebrew maintainers had our first in-person meeting at and around the [FOSDEM 2019 conference](https://fosdem.org/2019/) in Brussels. Maintainers travelled from as far as India and Canada in order to get face-time with each other and have high-bandwidth conversations.
```
Comment (reitermarkus) [inline _posts/2019-06-14-homebrew-maintainer-meeting.md]: “we Homebrew maintainers”, or maybe just “we”

File: _posts/2019-06-14-homebrew-maintainer-meeting.md (hunk 2 above)
Comment (reitermarkus) [reply _posts/2019-06-14-homebrew-maintainer-meeting.md]: Also, the last `]` is a typo.

**Comment by MikeMcQuaid** [comment]
Thanks @vitorgalvao and @reitermarkus for the review comments and everyone else who looked this over.

**Pull request merged**